pom.xml

```diff
@@ -1445,31 +1445,20 @@
                 <inherited>true</inherited>
                 <extensions>true</extensions>
                 <executions>
-                    <execution>
-                        <id>versions</id>
-                        <phase>validate</phase>
-                        <goals>
-                            <goal>cleanVersions</goal>
-                        </goals>
-                        <configuration>
-                            <versions>
-                                <sshd.osgi.version.clean>${project.version}</sshd.osgi.version.clean>
-                            </versions>
-                        </configuration>
-                    </execution>
                     <execution>
                         <id>bundle-manifest</id>
                         <phase>process-classes</phase>
                         <goals>
                             <goal>manifest</goal>
                         </goals>
                         <configuration>
+                            <supportIncrementalBuild>true</supportIncrementalBuild>
                             <instructions>
                                 <Import-Package><![CDATA[
-                                  org.apache.sshd*;version="[$(version;==;${sshd.osgi.version.clean}),$(version;=+;${sshd.osgi.version.clean}))",
-                                  org.slf4j*;version="${range;[==,${slf4j.upper.bound})}",
+                                  org.apache.sshd*;version="$<range;[===,=+);$<maven_version;${project.version}>>",
+                                  org.slf4j*;version="$<range;[==,$<slf4j.upper.bound>)>",
```

Comment:
`${slf4j.upper.bound}`. That is a maven property, not a bnd macro!

Reply:
That's right. I did more than I wanted. But since all properties of the project are injected into the BND-Analyser that property would have been substituted there and the result would have been the same. Now it is again done by maven.